### PR TITLE
Prettier all liquid files [part 2]

### LINKED ISSUE #2054: Prettier all liquid files
We should format all liquid files in Dawn using the prettier liquid plugin. Formatting whole files as part of a feature pull request is disruptive to the review process so it's probably best this is done intentionally and independently. 

A case could probably be made that it would be safer to do this for each file individually so we can better verify no unintentional changes are introduced by quirks in the plugin. But we can also see what the scope of the formatting changes are and determine if we feel that's necessary.

### PR DESCRIPTION
**Pull request by eugenekasimov**
### PR Summary: 

In this PR I use Prettier plugin to format **almost all** liquid files for **sections**. 

I didn't touch **main-product**, **header**, **footer**, and **featured-product**. I assume the most of the changes over the latest period were there. Even though there shouldn't be any issue, I believe it will help avoid mess during the upcoming rebases.

The idea is to run the Prettier plugin on all liquid files in Dawn. After that all devs will start using the plugin for default. That will allow us to avoid seeing changes that were made by formatting in PRs.

### Why are these changes introduced?

Fixes #2054

### Visual impact on existing themes

No visual impact.

### Demo links

- [Editor](https://os2-demo.myshopify.com/admin/themes/137449799702/editor)

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [ ] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by ludoboludo — COMMENTED** [review]
Just a few notes, you can probably set the PR to `Ready for review` as well

File: sections/collage.liquid [hunk 1]
```diff
@@ -24,18 +24,38 @@
...
+                    {%- capture sizes -%}
+                      (min-width: {{ settings.page_width }}px)
+                      {% if section.blocks.size == 1 -%}
+                        calc({{ settings.page_width }}px - 100px)
+                      {%- else -%}
+                        {{- settings.page_width | minus: 100 | times: 0.67 | round }}px
+                      {%- endif -%}
+                      , (min-width: 750px)
+                      {%- if section.blocks.size == 1 %} calc(100vw - 100px){% else %} 500px{% endif -%}
+                      , calc(100vw - 30px)}
```
[the remark this answers is not in the document]
Comment (charlespwd) [reply sections/collage.liquid:52]: Oh shoot, so `capture` is something that I fixed but haven't released yet. I'm not 100% confident it's OK in 0.4.

https://github.com/Shopify/prettier-plugin-liquid/pull/114

File: sections/main-collection-banner.liquid [hunk 2]
```diff
@@ -4,7 +4,7 @@
 {%- style -%}
   @media screen and (max-width: 749px) {
     .collection-hero--with-image .collection-hero__inner {
-      padding-bottom: calc({{ settings.media_shadow_vertical_offset | at_least: 0 }}px + 2rem);
+      padding-bottom: calc({{settings.media_shadow_vertical_offset | at_least: 0}}px + 2rem);
```
[the remark this answers is not in the document]
Comment (eugenekasimov) [reply sections/main-collection-banner.liquid:7]: Hey @charlespwd we noticed a case when whitespace was removed 👆 when we didn't expect it. Is it something that makes sence for you or you want me to open an issue for this case?

File: sections/product-recommendations.liquid [hunk 3]
```diff
@@ -1,6 +1,11 @@
 <link rel="stylesheet" href="{{ 'component-card.css' | asset_url }}" media="print" onload="this.media='all'">
 <link rel="stylesheet" href="{{ 'component-price.css' | asset_url }}" media="print" onload="this.media='all'">
-<link rel="stylesheet" href="{{ 'section-product-recommendations.css' | asset_url }}" media="print" onload="this.media='all'">
+<link
+  rel="stylesheet"
+  href="{{ 'section-product-recommendations.css' | asset_url }}"
+  media="print"
+  onload="this.media='all'"
+>
```
[the remark this answers is not in the document]
Comment (ludoboludo) [reply sections/product-recommendations.liquid:8]: @eugenekasimov maybe we can all have that enabled ^ as I don't think we need to make our asset `link`s take too much space 🤔 
Any thoughts ?

File: sections/main-collection-banner.liquid (hunk 2 above)
[the remark this answers is not in the document]
Comment (charlespwd) [reply sections/main-collection-banner.liquid:7]: I opened https://github.com/Shopify/prettier-plugin-liquid/issues/119 in response.

[the remark this answers is not in the document]
Comment (ludoboludo) [reply sections/main-collection-banner.liquid:7]: odd. maybe we need to check the `settings.json` file to confirm Eugene's settings ?

[the remark this answers is not in the document]
Comment (eugenekasimov) [reply sections/main-collection-banner.liquid]: Okay, I have updated this page. I'm not sure what caused this issue but it's not happening now. I repeated a few times and it seems working fine.

**Review by ludoboludo — APPROVED** [review]

**Pull request merged**